### PR TITLE
Update catalog docs with rancher v2.2.0 new features

### PR DESCRIPTION
**Pull request by guangbochen**
**Description:**
Update catalog docs with rancher v2.2.0 new features

**Related Issues:**
#1167

### CODE REVIEW

**Comment by loganhz** [comment]
LGTM

File: content/rancher/v2.x/en/catalog/_index.md [hunk 1]
```diff
@@ -44,15 +44,28 @@ Within Rancher, there are default catalogs packaged as part of Rancher. These ca
...
+The Git URL needs to be one that `git clone` [can handle](https://git-scm.com/docs/git-clone#_git_urls_a_id_urls_a) and must end in `.git`. The branch name must be a branch that is in your catalog URL. If no branch name is provided, it will use the `master` branch by default. Whenever you add a catalog to Rancher, it will be available immediately.
+
+
+#### Add Custom Helm Chart Repositories
+
+A Helm chart repository is an HTTP server that houses one or more packaged charts. Any HTTP server that can serve YAML files and tar files and can answer GET requests can be used as a repository server.
+
+Helm comes with built-in package server for developer testing (helm serve). The Helm team has tested other servers, including Google Cloud Storage with website mode enabled, S3 with website mode enabled or hosting custom chart repository server using open-source projects like [ChartMuseum](https://github.com/helm/chartmuseum).
+
+In Rancher, You can add the custom Helm chart repository with only specify a catalog name and the URL address of the chart repository.
```
Comment (ankur111) [inline content/rancher/v2.x/en/catalog/_index.md]: "In Rancher, You can add the custom Helm chart repository with only specify a catalog name" - this doesn't sound right. 
Is it "In Rancher, you can add the custom Helm chart repository with only a catalog name..."

File: content/rancher/v2.x/en/catalog/_index.md [hunk 2]
```diff
@@ -44,15 +44,28 @@ Within Rancher, there are default catalogs packaged as part of Rancher. These ca
...
+
+#### Add Private Git/Helm Chart Repositories
+_Available as of v2.2.0_
+
+From Rancher v2.2.0, you can now select to add private catalog repositories using credentials like Username and Password, you may also want to use the
```
Comment (ankur111) [inline content/rancher/v2.x/en/catalog/_index.md]: how about "In Rancher v2.2.0, you can add private..... "
"You may also want to use the "

File: content/rancher/v2.x/en/catalog/chart-custom-values/_index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+need to be translated into `outer.inner=value`, `servers[0].port=80` and `servers[0].host=example` respectively in the Answers section
+(Read more about [Format of Custom Values Using `--Set`](https://github.com/helm/helm/blob/master/docs/using_helm.md#the-format-and-limitations-of---set)).
+
+![dot-custom-values-form.png]({{< baseurl >}}/img/rancher/catalog/dot-custom-values-form.png)
+or through `Edit as YAML` view:
+![dot-custom-values.png]({{< baseurl >}}/img/rancher/catalog/dot-custom-values.png)
+<hr>
+
+### Supports Real Custom `values.yaml` for Catalog Apps
```
Comment (ankur111) [inline content/rancher/v2.x/en/catalog/chart-custom-values/_index.md]: This title "Supports Real Custom......" doesn't sound right.
Perhaps it should be "Support For Real Custom...."

**Comment by loganhz** [comment]
updated

**Pull request merged**